package/python-kiwi-spec-template

```diff
@@ -374,6 +374,7 @@ Requires:       python%{python3_pkgversion}-docopt
 Requires:       python%{python3_pkgversion}-lxml
 Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-setuptools
+Requires:       (python%{python3_pkgversion}-typing_extensions if python%{python3_pkgversion} < 3.8)
```

Comment:
The package name is `python3-typing-extensions` in RH/Fedora and Debian/Ubuntu.

And conditional dependencies don't work on Debian/Ubuntu at all, so this needs to be handled correctly for that too.